doc/source/user_guide/external_storage.rst

```diff
@@ -0,0 +1,12 @@
+.. _external_storage_index:
```

Comment:
I think this page should be placed under ``Deployment -> Kubernetes deployment``, and also mentioned in ``Cloud deployment``.